content/ember-cli/v5/travis-ci-support.md

```diff
@@ -0,0 +1,8 @@
+---
+id: travis-ci-support
```

Comment:
the error is in the test log: `You can't define an ID for a deprecation. If you need an ID that is different from the file name then use displayId` delete this id 👍